skabase/SKABaseDevice/SKABaseDevice.xmi

```diff
@@ -34,6 +34,22 @@
       <status abstract="false" inherited="false" concrete="true" concreteHere="true"/>
       <DefaultPropValue>localhost</DefaultPropValue>
     </deviceProperties>
+    <deviceProperties name="CentralLogger" description="">
+      <type xsi:type="pogoDsl:StringType"/>
+      <status abstract="false" inherited="false" concrete="true" concreteHere="true"/>
+    </deviceProperties>
+    <deviceProperties name="ElementLogger" description="">
+      <type xsi:type="pogoDsl:StringType"/>
+      <status abstract="false" inherited="false" concrete="true" concreteHere="true"/>
+    </deviceProperties>
+    <deviceProperties name="ElementLoglevel" description="">
+      <type xsi:type="pogoDsl:UShortType"/>
+      <status abstract="false" inherited="false" concrete="true" concreteHere="true"/>
+    </deviceProperties>
+    <deviceProperties name="ServerHostName" description="">
+      <type xsi:type="pogoDsl:StringType"/>
+      <status abstract="false" inherited="false" concrete="true" concreteHere="true"/>
+    </deviceProperties>
```

Comment:
If the SKABaseDevice.xmi file changes, then all the derived XMI files need to change, and all the Python files generated from them also need to change!  There are tools to do this regeneration automatically.  However, we should first reach agreement on the changes in the base file before going through that effort.

Reply:
Removing it.